SocketIOClientSwift/SocketTypes.swift

```diff
@@ -27,8 +27,9 @@ import Foundation
 public typealias AckCallback = ([AnyObject]) -> Void
 public typealias NormalCallback = ([AnyObject], SocketAckEmitter?) -> Void
 public typealias OnAckCallback = (timeoutAfter: UInt64, callback: AckCallback) -> Void
+public typealias SocketOptionsSet = Set<SocketIOClientOption>
```

Comment:
I this a clear solution.
Because I don't see at first sight that this should be a Set.